src/components/search/index.ts

```diff
@@ -1,3 +1,5 @@
-import Search from './search';
+import SearchResults from './results';
+import SearchBox from './search-box';
+import SortBox from './sort-box';
 
-export default Search;
+export { SearchResults, SearchBox, SortBox };
```

Comment:
We definitely needed to do this, as the search becomes more complex. Great work!